pages/alp/patterns-page.ts

```diff
@@ -0,0 +1,35 @@
+import { expect, type Locator, type Page } from '@playwright/test';
+
+export class PatternsPage {
```

Comment:
We could try to use "extends" here, but we still need two different tests for each products, because of the "import" section.